src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -4,35 +4,28 @@
   <PropertyGroup>
```

Comment:
I thought the plan was to keep this project and make a new ".NETCore" csproj file so we could build it for either target.